pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -223,7 +230,7 @@ func (r *InferenceServiceReconciler) updateStatus(desiredService *v1beta1api.Inf
 		// This is important because the copy we loaded from the informer's
```

Comment:
Not sure whether the DeepEqual comparison above should be reduced in scope to not look at the parts of the status that the modelmesh controller "owns" in the case that this is a modelmesh deployment?